code/__HELPERS/text.dm

```diff
@@ -136,8 +136,8 @@
 				number_of_alphanumeric++
 				last_char_group = 3
 
-			// '  -  .
-			if(39,45,46)			//Common name punctuation
+			// '  -  . ,
+			if(39,45,46, 44)			//Common name punctuation

```

Comment:
Does this allow , in ALL player names, not just Kidan ones?

Reply:
Yes. Name code can't currently tell the difference. This also means that characters intended for AI names are also all allowed in Player names. You can put stuff like ! and @ in your character name if you want (though it's going to get you bwoinked I'd expect)

Reply:
I think it would be better to simply not allow , in any character names.
But, that's just my preference.
Yes, it would negate the whole "X, Y of Z" naming scheme, but frankly I'd prefer a "X Y" or "X of Y" scheme anyway, without a comma.

Reply:
Take it up with the lore team (If you can find them, that is). More seriously...do we have any issues with silly names containing special characters? I frankly never seen them in the round.

Reply:
I don't see any problem with `,` being allowable in character names.

Reply:
Inconsistent spacing, please add spaces after the rest of those commas